UPGRADE.md

```diff
@@ -8,6 +8,10 @@ awareness about deprecated code.
 
 # Upgrade to 4.0
 
+## BC BREAK: removed `ConsolerRunner`.
+
+The `ConsoleRunner` class has been removed.
```

Comment:
We should merge this with the section "Removed the `doctrine-dbal` binary." because we mention the removal of `ConsolerRunner` methods there.